src/main/jhipster/src/main/webapp/app/shared/user-profile/user-profile.module.ts

```diff
@@ -0,0 +1,26 @@
+import { NgModule } from '@angular/core';
+import { RouterModule } from '@angular/router';
+import { USER_PROFILE_ROUTER } from './user-profile.route';
+import { UserProfileUpdateDialogComponent, UserProfileUpdatePopupComponent } from './user-profile-update-dialog.component';
+import { UserProfileServices } from './service/user-profile-services.service';
+import { BmsjHipsterSharedModule } from '..';
+
+@NgModule({
+    imports: [
+        BmsjHipsterSharedModule,
+        RouterModule.forChild(USER_PROFILE_ROUTER)
+    ],
+    declarations: [
+        UserProfileUpdateDialogComponent,
+        UserProfileUpdatePopupComponent,
+    ],
+    entryComponents: [
+        UserProfileUpdateDialogComponent,
+        UserProfileUpdatePopupComponent,
+    ],
+    providers: [
+        UserProfileServices
+    ]
+})
+export class UserProfileModule {
```

Comment:
Maybe a better folder would be `/app/entities/user-profile` as it is not a reusable component